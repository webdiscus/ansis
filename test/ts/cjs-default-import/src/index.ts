import ansis from 'ansis';

const message: string = `Hello, ${ansis.red`World`}!`;
console.log(message);