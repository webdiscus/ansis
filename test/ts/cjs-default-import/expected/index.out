Hello, [31mWorld[39m!
